litgpt/config.py

```diff
@@ -91,6 +91,16 @@ def __post_init__(self):
 
         self.rope_n_elem = int(self.rotary_percentage * self.head_size)
 
+    @classmethod
+    def get_name_from_hf_config(cls, hf_name: str) -> str:
```

Comment:
1) How about implementing this in `from_name`? I don't see the need to duplicate the functionality. 
2) The class is not used, so this isn't really a classmethod

Reply:
Thanks, I merged these two